ups/product_deps

```diff
@@ -326,22 +332,9 @@ end_qualifier_list
 # be copied to the table file verbatim.
 table_fragment_begin
 
-#    envSet (MU2E_BASE_RELEASE, \${OFFLINE_DIR} )
-#    envSet (MU2E_SEARCH_PATH, \${OFFLINE_DIR}/config:\${MU2E_DATA_PATH}:/cvmfs/mu2e.opensciencegrid.org/DataFiles )
-#    envSet (FHICL_FILE_PATH, \${OFFLINE_DIR}/config:\${OFFLINE_DIR}/config/Offline/fcl )
-#    envSet (OFFLINE_VERSION, \${UPS_PROD_VERSION} )
-#    pathAppend( ROOT_INCLUDE_PATH, \${OFFLINE_INC})
-
-#    exeActionRequired(GetFQDir)
-#    envSet (OFFLINE_LIB, \${OFFLINE_DIR}/\${OFFLINE_FQ}/lib )
-#    pathAppend( CET_PLUGIN_PATH, \${OFFLINE_LIB} )
-#    pathPrepend(PATH, "\${OFFLINE_DIR}/\${OFFLINE_FQ}/bin")
-
-#    if ( test \`uname\` = "Darwin" )
-#      pathPrepend(DYLD_LIBRARY_PATH, \${\${UPS_PROD_NAME_UC}_LIB})
-#    else()
-#      pathPrepend(LD_LIBRARY_PATH, \${\${UPS_PROD_NAME_UC}_LIB})
-#    endif ( test \`uname\` = "Darwin" )
+envSet (MU2E_BASE_RELEASE, \${OFFLINE_DIR} )
```

Comment:
@rlc I believe that MU2E_BASE_RELEASE is obsolete and no longer used.  I don't see the string anywhere in Offline, muse, museconfig or mu2egrid. 

@eflumerf if Ray confirms, please remove this line.

Reply:
I believe this is OK, but its hard to be completely sure.  I think you can go ahead